test/commands/summary.spec.js

```diff
@@ -15,20 +15,19 @@ test("summary should execute summary command on fixtures 'result-test1.json'", a
   const lines = [
     /Global Stats: express.*$/,
     /.*/,
-    /Total of packages:.*65.*$/,
+    /Total of packages:.*64.*$/,
     /Total size:.*1.62 MB.*$/,
-    /Packages with indirect dependencies:.*6.*$/,
+    /Packages with indirect dependencies:.*5.*$/,
     /.*/,
     /Extensions:.*$/,
-    /\(48\) {2}- \(50\) \.md - \(50\) \.json - \(50\) \.js - \(5\) \.ts - \(2\) \.yml.*$/,
+    /\(48\) {2}- \(50\) \.md - \(50\) \.js - \(50\) \.json - \(2\) \.yml - \(5\) \.ts.*$/,
     /.*/,
     /Licenses:.*$/,
     /\(47\) MIT - \(2\) ISC.*$/,
     /.*/
   ];
-  tape.plan(lines.length * 2);
```

Comment:
It's complicated to keep `tape.plan` in these tests as the number of lines differs regarding the node version. 

Another approach could be to create a helper that asserts that a set of lines exists in the stream regardless of the number of lines.